v0/trigger/webhook_trigger/harness_pr_spec.yaml

```diff
@@ -8,13 +8,11 @@ allOf:
         items:
           type: string
           enum:
+            - Created
```

Comment:
leaving a comment here as reminder to update values with latest changes in harness-core PR